app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Your Dashboard</h1>
+
+<% @jobs.each do |job| %>
+  <ol><%= job.title %> <%= job.level_of_interest %></ol>
+<% end %>
+
+<h1>Top Three Companies by Level of Interest</h1>
+<% @companies.each do |company| %>
+  <ol><%= company.name %> <%= company.jobs.average(:level_of_interest).round(1) %></ol>
```

Comment:
this averaging should be done in a method in a specific class. it doesn't belong in your view. i suggest creating a class method in your `Job` class 👍